schema_based_on_diagram

```diff
@@ -0,0 +1,50 @@
+CREATE TABLE patients(
+    id INTEGER NOT NULL,
+    name VARCHAR(150) NOT NULL,
+    date_of_birth DATE NOT NULL,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories(
+    id INTEGER NOT NULL,
+    admitted_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    patient_id INTEGER NOT NULL,
+    status VARCHAR(150) NOT NULL,
+    PRIMARY KEY(id),
+    CONSTRAINT medical_histories_patient_id_foreign FOREIGN KEY(patient_id) REFERENCES patients(id)
+);
+
+CREATE TABLE invoices(
+    id INTEGER NOT NULL,
+    total_amount DECIMAL(8, 2) NOT NULL,
+    generated_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    payed_at TIMESTAMP(0) WITHOUT TIME ZONE NOT NULL,
+    medical_history_id INTEGER NOT NULL,
+    PRIMARY KEY(id),
+    CONSTRAINT invoices_medical_history_id_foreign FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id)
+);
+
+CREATE TABLE treatments(
+id INTEGER NOT NULL,
+type VARCHAR(150) NOT NULL,
+name VARCHAR(150) NOT NULL,
+PRIMARY KEY(id)
+);
+
+CREATE TABLE medical_histories_treatments(
+medical_histories_id INTEGER NOT NULL,
+treatment_id INTEGER NOT NULL,
```

Comment:
- [ ] In this join table for many-to-many relationship, all FOREIGN KEYS are set by using `refrences table_name(id)` in column definition. This implies that for line 35 and 36, you should have something like below:
- `medical_history_id int refrences medical_histories(id),`
- `treatment_id int refrences treatments(id),`
Please also verify the column name, if it should be `medical_histories_id `or `medical_history_id`